app/static/views/yara_rule/yara_rules.html

```diff
@@ -603,6 +603,14 @@ <h4 class="modal-title" id="myC2ipLabel" style="float: left">
                                     </option>
                                 </select>
                             </div>
+                            <div style="max-width: min-content">
+                                <label>&nbsp;</label>
+                                <button type="button" class="btn btn-danger"
+                                        ng-disabled="!selectedRevisions.compared"
+                                        confirmed-click="revertRevision(yara_rule.id, selectedRevisions.compared.revision)"
+                                        ng-confirm-click="Are you sure you want to revert to revision?">Revert
```

Comment:
I wanted to specify the revision number in the `ng-confirm-click` dialog, but I couldn't get it to display.